plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestTrinoGlueCatalog.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import com.amazonaws.services.glue.AWSGlueAsyncClientBuilder;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.airlift.log.Logger;
+import io.trino.plugin.hive.HdfsConfig;
+import io.trino.plugin.hive.HdfsConfigurationInitializer;
+import io.trino.plugin.hive.HdfsEnvironment;
+import io.trino.plugin.hive.HiveHdfsConfiguration;
+import io.trino.plugin.hive.authentication.NoHdfsAuthentication;
+import io.trino.plugin.hive.metastore.glue.GlueMetastoreStats;
+import io.trino.plugin.iceberg.catalog.TrinoCatalog;
+import io.trino.plugin.iceberg.catalog.glue.GlueIcebergTableOperationsProvider;
+import io.trino.plugin.iceberg.catalog.glue.TrinoGlueCatalog;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.security.PrincipalType;
+import io.trino.spi.security.TrinoPrincipal;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import static io.trino.testing.TestingConnectorSession.SESSION;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static org.testng.Assert.assertEquals;
+
+public class TestTrinoGlueCatalog
+        extends BaseTestTrinoCatalog
+{
+    private static final Logger LOG = Logger.get(TestTrinoGlueCatalog.class);
+
+    @Override
+    protected TrinoCatalog createTrinoCatalog(boolean useUniqueTableLocations)
+    {
+        HdfsEnvironment hdfsEnvironment = new HdfsEnvironment(new HiveHdfsConfiguration(
+                new HdfsConfigurationInitializer(
+                        new HdfsConfig(),
+                        ImmutableSet.of()),
+                ImmutableSet.of()),
+                new HdfsConfig(),
+                new NoHdfsAuthentication());
+        return new TrinoGlueCatalog(
+                hdfsEnvironment,
+                new GlueIcebergTableOperationsProvider(new HdfsFileIoProvider(hdfsEnvironment)),
+                AWSGlueAsyncClientBuilder.defaultClient(),
+                new GlueMetastoreStats(),
+                Optional.empty(),
+                Optional.empty(),
+                useUniqueTableLocations);
+    }
+
+    @Test
+    public void testDefaultLocation()
```

Comment:
Can this be in `BaseTestTrinoCatalog`?

Reply:
This tests the `hive.metastore.glue.default-warehouse-dir` config, which is Glue specific